datalad/distributed/create_sibling_ghlike.py

```diff
@@ -458,6 +467,7 @@ def repo_create_request(self, reponame, organization, private,
         headers = self.request_headers
 
         if dry_run:
+            res_msg = "would create {} sibling and repository named {}"
```

Comment:
I am a bit confused. Where is `res_msg` used?

Reply:
Oops, well spotted! I accidentally left and committed part of my first attempt at the message. This line should go away.
```suggestion
```